frontend/src/components/PropertyCard.vue

```diff
@@ -359,13 +367,47 @@ export default {
       return Math.min(Math.max(result, 10), 90);
     },
   },
-
+  watch: {
+    coordinates() {
+      this.fetchProperty();
+    },
+  },
+  created() {
+    this.fetchProperty();
+  },
   methods: {
     capitalizeWords,
 
     onHide() {
       this.$emit("hide");
     },
+
+    fetchProperty() {
+      axios({
+        url: "http://localhost:8282/graphql",
```

Comment:
Perhaps it might be helpful to outsource the Axios URL to a separate file, so we can replace it later if running on non-localhost